inspire/modules/workflows/tasks/submission.py

```diff
@@ -91,6 +91,7 @@ def _create_ticket(obj, eng):
             email=email,
             title=d.title,
             identifier=sip.metadata.get("system_number_external", {}).get("value", ""),
+            user_comment=sip.metadata.get("hidden_note", {}).get("value", ""),
```

Comment:
Yes to both. In that way the field will always be a list and ```sip.metadata.get("hidden_note", [])``` will be the right thing to do.